sdk/javascript/src/nem/models.js

```diff
@@ -1677,7 +1677,7 @@ class MosaicDefinition {
 		size += arrayHelpers.size(this.properties);
 		size += 4;
 		if (0 !== this.levySize)
-			size += (undefined === this.levy || null === this.levy) ? 0 : this.levy.size;
+			size += this.levy.size;
```

Comment:
this (levySize + levy) _could_ have been potentially one of the problematic places, so if it works, we're probably missing some tests, BUT

if this should be handled differently in the future it's clearly not in the scope of this change :)